packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -185,6 +193,13 @@ declare global {
        * @param value - the value which needs to be write to form input.
        */
       text(formId: string, attributeName: string, value: string | number): Chainable<void>;
+      /**
+       * Clear all selected items from the form input.
+       * @category Data removing
+       *
+       * @param className - The ID of class which contain form input.
+       */
+      clearSelectedItems(className: string): Chainable<void>;
```

Comment:
Could this method be bound to the input's editing ID? I am thinking of a case, where multiple inputs will be of list type (items could be added by typing and pressing enter, and then being shown under the input) and IMHO this would erase them all. I also suggest renaming it to something like `clearListAttributeItems`